docs/usage/shoot_credentials_rotation.md

```diff
@@ -170,9 +171,50 @@ It is currently **not** rotated automatically and there is no way to trigger it
 
 ## `ServiceAccount` Token Signing Key
```

Comment:
There are no integration/e2e tests added as part of this PR for the new feature (which can be acceptable having in mind that it is behind alpha feature gate). Can you create new issue (or new item in umbrella issue) to add such tests?

Reply:
We will add the etcd encryption secret rotation as well and then refactor the existing e2e test which only covers CA rotation today as part of https://github.com/gardener/gardener/issues/3292.
Eventually, we envision having only one e2e test suite for the credentials rotation.